src/mimeparser.rs

```diff
@@ -1398,6 +1398,17 @@ impl MimeMessage {
             } else {
                 Viewtype::File
             }
+        } else if msg_type == Viewtype::Image || msg_type == Viewtype::Gif {
```

Comment:
`Viewtype::Sticker` should be added as well, which is basically an image containing fully transparent pixels in Delta Chat